src/main/kotlin/no/nav/tilleggsstonader/sak/journalføring/JournalføringService.kt

```diff
@@ -136,6 +150,29 @@ class JournalføringService(
         )
     }
 
+    fun journalførTilNyKlage(
+        journalpostId: String,
+        personIdent: String,
+        stønadstype: Stønadstype,
+        behandlingÅrsak: BehandlingÅrsak,
+        oppgaveBeskrivelse: String? = null,
+        journalførendeEnhet: String,
+        dokumentTitler: Map<String, String>? = null,
+        logiskVedlegg: Map<String, List<LogiskVedlegg>>? = null,
+    ) {
+        val journalpost = journalpostService.hentJournalpost(journalpostId)
+
+        val fagsak = hentEllerOpprettFagsakIEgenTransaksjon(personIdent, stønadstype)
+
+        validerKanOppretteBehandling(journalpost, fagsak, behandlingÅrsak, gjelderKlage = true)
+
+        feilHvis(journalpost.harStrukturertSøknad()) { "" }
```

Comment:
Hva er forventet her?

Reply:
Klagebehandlingen skal ikke opprettes dersom Journalposten har dokumenter med brevkode:
```
BARNETILSYN("NAV 11-12.15")
LÆREMIDLER("NAV 11-12.16")
```

Reply:
Dette for å unngå at man oppretter en Klagebehandling når man gjør en Journalføring på en behandling som er enten for BARNETILSYN eller LÆREMIDLER

Reply:
Har lagt inn en kommentar her: de3b6d1b8fb2945f059cea675f7db9c6d3796571